specification/riskiq/Easm/models.tsp

```diff
@@ -0,0 +1,1635 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Easm;
+
+alias AssetsListTrait = QueryParametersTrait<{
+    ...FilterQueryParameter;
+    ...OrderByParameter;
+    ...SkipQueryParameter;
+    ...MaxPageSizeQueryParameter;
+    ...MarkParameter;
+}>;
+alias AssetsUpdateParameter = {
+    ...RequiredFilterParameter;
+};
+alias DataConnectionsListTrait = QueryParametersTrait<{
+    ...SkipQueryParameter;
+    ...MaxPageSizeQueryParameter;
+}>;
+alias DiscoveryGroupsListTrait = QueryParametersTrait<{
+    ...FilterQueryParameter;
+    ...SkipQueryParameter;
+    ...MaxPageSizeQueryParameter;
+}>;
+alias DiscoveryGroupsListRunsParameter = {
+    ...FilterQueryParameter;
+    ...SkipQueryParameter;
+    ...MaxPageSizeQueryParameter;
+    ...DiscoGroupNameParameter;
+};
+alias DiscoveryTemplatesListTrait = QueryParametersTrait<{
+    ...FilterQueryParameter;
+    ...SkipQueryParameter;
+    ...MaxPageSizeQueryParameter;
+}>;
+alias SavedFiltersListTrait = QueryParametersTrait<{
```

Comment:
This alias seems to be have the same parameters as the one on line 26 and 37, I'd suggest just defining one with a generic name and reusing it on the corresponding routes. Same feedback for the others.